src/App/component/IssuesPage.jsx

```diff
@@ -100,32 +100,56 @@ function IssuesPage(prop) {
   }, [currentProject, prop.startMonth, prop.endMonth])
 
   useEffect(() => {
-    const { endMonth } = prop
-    let chartDataset = { labels: [], data: { created: [], closed: [] } }
-    let issueListDataSortedByCreatedAt = issueListData
-    let issueListDataSortedByClosedAt = issueListData
+    const chartDataset = generateIssueDataSet()
+    setDataForIssueChart(chartDataset)
+  }, [issueListData])
 
-    issueListDataSortedByCreatedAt.sort((a, b) => a.createdAt - b.createdAt)
-    issueListDataSortedByClosedAt.sort((a, b) => a.closedAt - b.closedAt)
+  const generateIssueDataSet = () => {
+    const { startMonth, endMonth } = prop
+    let chartDataset = { labels: [], data: { closed: [] , created: []} }
```

Comment:
Suggest make this to be a constant.
```suggestion
    const chartDataset = { labels: [], data: { closed: [] , created: []} }
```